### PR TITLE
Update to ReadMe to include deploy instructions

### PR DESCRIPTION
**Pull request by kswenson**
Add section on deploy instructions, including CloudFront invalidation instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,30 +85,137 @@ Place the following in your `/etc/apache2/extra/httpd-vhosts.conf` file (replaci
...
+./build_and_deploy.sh staging_s3
+```
+
+This can take several minutes. If everything completes successfully, the script will end with a message like:
+```
+Public URL of the object is: http://geniverse-lab.concord.org.s3.amazonaws.com/staging/index.html
+Done. Don't forget to invalidate the index files in cloudfront!
+```
+
+Ultimately, the new deployment can be tested at http://geniverse-lab.concord.org/staging. The warning about invalidating the index files is because deployed changes won't be visible immediately. "CloudFront takes up to 24 hours to clear cache. If you need to hurry it, it can be cleaned manually, though we pay a (trivial) amount when we do that, so I try not to unless we're demoing that day or something." Looking into the costs further, at the time of this writing Concord Consortium is allowed 1000 invalidations per month for free. Each invalidation beyond that is $.005.
```
Comment (sfentress) [inline README.md]: Sounds like we don't even need the line about trying not to do it, then.

File: README.md [hunk 2]
```diff
@@ -75,30 +85,137 @@ Place the following in your `/etc/apache2/extra/httpd-vhosts.conf` file (replaci
...
+
+## Deploying Geniverse
+
+### Installing and Configuring s3cmd Tools
+
+Download `s3cmd` from http://s3tools.org/download and follow the included installation instructions.
+
+Run `s3cmd --configure` to configure it. You will need your AWS account credentials (Access Key and Secret Access Key).
+
+### Installing Gems
```
Comment (sfentress) [inline README.md]: Any reason why this is in the Deploying section? Won't they already have encountered this when running?

Comment (kswenson) [inline README.md]: If you had actually followed the instructions as they were (and as I did while writing the instructions), you would have installed the SproutCore gem manually during the initial installation and would only have encountered the need for the other gems when trying to run the deployment script. That said, it does make sense to mention it earlier so I moved it up and added a comment in the deployment section pointing back to the earlier gem installation section.

File: README.md (hunk 1 above)
Comment (kswenson) [inline README.md]: Done

**Pull request merged**